src/content/docs/dropins/order/tutorials/order-cancellation.mdx

```diff
@@ -0,0 +1,141 @@
+---
+
+import Diagram from '@components/Diagram.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+
+The Order dropin allows both logged in and guest users to cancel an order.
+
+In this tutorial, we will show:
+- the requirements to be able to cancel an order,
+- the cancellation workflow, and
+- the queries and mutations involved in the order cancellation process.
+
+## Requirements
+
+To be able to cancel an order, you need to have the configuration enabled in the backend.
+You can find it in the backoffice under *Stores > Configuration > Sales > Sales > Order Cancellation*.
+
+There are the following configurations:
+  <OptionsTable
+    compact
+    options={[
+    ['Configuration', 'Description'],
+    ['Order cancellation through GraphQL', 'It is a boolean. When it is set to `true`, the order cancellation will be enabled.'],
+    ['Order cancellation reasons', 'It is an array of reasons that can be selected by the user when cancelling an order.'],
+    ]}
+  />
+
+:::note[Note]
+Note: There is a default list of reasons, but you can customise according to your needs.
+:::
+
+<Diagram caption="The order cancellation configurations you can customise in the backoffice">
+  ![Cancellation configurations in the backoffice](@images/dropins/order/requirements-cancellation-configs.png)
+</Diagram>
+
+## Workflow
+
+Generally speaking, the workflow for order cancellation is as follows:
+- The user selects an order to cancel.
+- Submits the cancellation form after selecting a cancellation reason.
+- In case it is a logged-in user, the order is `Canceled` immediately.
+- But if it is a guest user, an email is sent with a link to confirm the cancellation. Once the user clicks the link, the order is `Canceled`.
+
+Let's review more in detail the workflow for a guest order cancellation:
+
+### Select an order to cancel
+
+The guest user provides the details to load an order to cancel:
+
+<Diagram caption="Form presented to the guest user to load order details.">
+  ![Guest user provides order details](@images/dropins/order/workflow-select-order-to-cancel.png)
+</Diagram>
+
+### Submit the form
+
+The guest user is able to select a cancellation reason and submit the form:
+
+<Diagram caption="Form presented to the guest user with different cancellation reasons.">
+  ![Guest user selects a cancellation reason](@images/dropins/order/workflow-select-cancellation-reason.png)
+</Diagram>
+
+After submitting the form, the changes are reflected on the *Order Status* page:
+
+<Diagram caption="Guest user is notified that cancellation has been requested.">
+  ![Cancellation has been requested](@images/dropins/order/workflow-cancellation-requested.png)
+</Diagram>
+
+### Confirm order cancellation
+
+The guest order remains in status `Pending` until the user clicks the link in the email to confirm the cancellation.
+After that, the order becomes `Canceled` and *Order Status* page is updated accordingly.
+
+## Queries & Mutations
```

Comment:
Even if we are not talking about how to do an order cancellation for logged-in users, I should add some comment and description about the cancelOrder mutation